Dual Battery Support.md

```diff
@@ -0,0 +1,262 @@
+#  Dual Battery Support
+
+macOS has buggy dual battery support and we won't have correct battery status without doing some changes.
```

Comment:
Buggy or no support?

Reply:
Buggy IMO,
If you leave the setup unimplemented, in about this mac > Power > it will show both batteries.
However it's totally bugged, no correct reporting, doesn't show both batteries on menu bar etc.
My guess is that long time ago in Yosemite/El Capitan era, they may have had plans for a Macbook or so with Dual batteries and start working on the software side but for any reason cancelled that and they may have not touched that part anymore, at least that's how i think.

Reply:
OK. I see that there are some SMC keys that have battery index, but not all.